01_login.php

```diff
@@ -54,134 +55,132 @@
+							$PW =  isset($_POST['password']) ?$_POST['password']: '';
 
-		</div>
-		<div id="id01" class="w3-modal">
-			<div class="w3-modal-content w3-card-4 w3-animate-zoom" style="max-width:600px">
+							//create a variable to store sql code for the 'Add Users' query
+							$insertquery = "INSERT INTO users( User_ID, Password ) VALUES( '$UserID','$PW' )";
 
-				<div class="w3-center"><br>
-					<span onclick="document.getElementById('id01').style.display='none'" class="w3-button w3-xlarge w3-transparent w3-display-topright" title="Close Modal">×</span>
-				</div>
+							if (mysqli_query($conn, $insertquery))
+							{
+								echo "<p1 class = white > Record inserted:</p1>";
+							}
+							else
+							{
+								echo "<p1 class = 'white'>Error inserting record:</p1>";
+							}
+							?>
 
-				<form class="w3-container" action="/action_page.php">
-					<div class="w3-section">
-						<label for = 'login'>Username:</label>
-						<input type = "text" name = "username" placeholder="Enter user name" style="width:100%" required/><br />
-						<label for = 'login'>Password:</label>
-						<input type = "password" name = "password" placeholder="Enter user password" style="width:100%" required/><br />
-						<a href="01_login.php" class="w3-button w3-block w3-green w3-section w3-padding" type="submit">Register </a>
+							<div class="w3-container w3-border-top w3-padding-16 w3-black">
+								<button onclick="document.getElementById('id01').style.display='none'" type="button" class="w3-right w3-button w3-red">Cancel</button>
+							</div>
 
+						</form>
 					</div>
+				</div>
+
+				<br>
+				<br>
 
-					<!-- add user php-->
-					<?php
-					//connect.php (tells where to connect servername, username, password, dbaseName)
-					require "communityproject_mysqli.php";
-					print "<p class = 'yellow'>Connected to server</p>";
-
-					$UserID = isset($_POST['username']) ?$_POST['username']: '';
-					$PW =  isset($_POST['password']) ?$_POST['password']: '';
-
-					//create a variable to store sql code for the 'Add Users' query
-					$insertquery = "INSERT INTO users( User_ID, Password ) VALUES( '$UserID','$PW' )";
-
-					if (mysqli_query($conn, $insertquery))
-					{
-						echo "<p class = 'white'>Record inserted:</p>";
-					}
-					else
-					{
-						echo "<p class = 'white'>Error inserting record:</p>";
-					}
-					?>
-
-					<div class="w3-container w3-border-top w3-padding-16 w3-light-grey">
-						<button onclick="document.getElementById('id01').style.display='none'" type="button" class="w3-right w3-button w3-red">Cancel</button>
-					</div>
 
-				</form>
 			</div>
 		</div>
 
-
 		<!-- Footer (sit on bottom of screen) -->
-		<footer class="w3-bottom w3-center w3-black w3-padding-64">
-			<p1>Created by <a target="_blank" class="w3-hover-text-orange">Soundwave</a> &copy; Copyright Liv Amer 2022</p1>
-
-
-
-
+		<footer class="w3-center w3-grey w3-padding-64">
```

Comment:
footer is made grey to match the navbar; creating a connection and a consistent theme